utils/yoke/derive/src/lib.rs

```diff
@@ -20,7 +20,7 @@ use synstructure::Structure;
 /// `zerovec::ZeroMap`.
 /// Please comment on https://github.com/unicode-org/icu4x/issues/844
 /// if you need this
-#[proc_macro_derive(Yokeable)]
+#[proc_macro_derive(Yokeable, attributes(yoke))]
```

Comment:
Please document the `yoke(prove...)` attribute  above, and remove the comment about it failing to compile.